### PR TITLE
allow larger ens size and non-zero start

### PR DESCRIPTION
**Pull request by allibco**
### Description of changes
Allows an ensemble of up to 2000 members.  Also allows you to specify an ensemble start number (this is so you could generate 100 members at a time, for example).

### Specific notes

Contributors other than yourself, if any:
just me

Fixes: [Github issue #s] And brief description of each issue.
N/A

User interface changes?: [ No/Yes ]
[ If yes, describe what changed, and steps taken to ensure backward compatibility ]
just a new option added (--es)

Testing performed (automated tests and/or manual tests):
testes

### CODE REVIEW

File: tools/statistical_ensemble_test/ensemble.py [hunk 1]
```diff
@@ -62,11 +85,22 @@ def main(argv):
...
         clone_count = ens_size - 1
-        if ens_size > 999:
-            print("Error: cannot have an ensemble size greater than 999.")
+        if ens_size > 2000:
+            print("Error: cannot have an ensemble size greater than 2000.")
             sys.exit()
         print("STATUS: ensemble size = " + str(ens_size))
 
+    #where to start ensemble                                                                                                                                                                                                                                                                           
+    start = opts_dict["es"]
```
Comment (jedwards4b) [inline tools/statistical_ensemble_test/ensemble.py:94]: maybe this should be ens_start for consistency and clarity

File: tools/statistical_ensemble_test/ensemble.py [hunk 2]
```diff
@@ -107,13 +144,18 @@ def main(argv):
...
+        for i in range(begin_i, clone_count + 1):  # 1: clone_count
             if run_type == "verify":
                 this_pertlim = get_pertlim_uf(rand_ints[i])
             else:  # full ensemble
                 this_pertlim = get_pertlim_uf(i)
 
-            iens = "{0:03d}".format(i)
+
+            #allow for 4 digit numbers
+            if i > 999:
```
Comment (jedwards4b) [inline tools/statistical_ensemble_test/ensemble.py:155]: Why not just always use 0:04d?

Comment (allibco) [reply tools/statistical_ensemble_test/ensemble.py:155]: I will change this

**Comment by allibco** [comment]
I made the requested changes. thanks!

**Review by jedwards4b — APPROVED** [review]

**Pull request merged**